app/components/page_list_component/view.html.erb

```diff
@@ -34,7 +34,7 @@
 
           <% if render_routing? %>
             <% page.conditions.each do |condition| %>
-              <div id="<%= PageListComponent::ErrorSummary::View.error_id(condition.id) %>" class="govuk-summary-list__row app-page-list__row <%= class_names( "govuk-form-group--error": condition.has_errors?) %>">
+              <div id="<%= PageListComponent::ErrorSummary::View.error_id(condition.id) %>" class="govuk-summary-list__row app-page-list__row <%= class_names( "govuk-form-group--error": condition.has_routing_errors) %>">
```

Comment:
```suggestion
              <div id="<%= PageListComponent::ErrorSummary::View.error_id(condition.id) %>" class="govuk-summary-list__row app-page-list__row <%= class_names( "govuk-form-group--error": condition.has_routing_errors?) %>">
```

Rails is clever enough when you add `?` to the method name to expect the value to be true/false if that is what is expected from this attribute.

Reply:
Nice!